src/tests/class_tests/openms/source/MSSpectrum_test.cpp

```diff
@@ -1451,6 +1451,62 @@ START_SECTION(([MSSpectrum::RTLess] bool operator()(const MSSpectrum &a, const M
 }
 END_SECTION
 
+START_SECTION((std::pair<DriftTimeUnit, std::vector<float>> maybeGetIMData() const))
+{
+  // Test successful retrieval of ion mobility data
+  MSSpectrum spec;
+  
+  // Create a float data array with ion mobility data
+  DataArrays::FloatDataArray im_array;
+  im_array.setName("Ion Mobility");
```

Comment:
For timsTOF data with proteowizard ion mobility is referred to as "inverse reduced ion mobility" Maybe check should also include this?

Reply:
Is this really the case, because I only see "mean inverse reduced ion mobility array" in our code
```
    /// get drift time array (may be null)
    BinaryDataArrayPtr getDriftTimeArray() const
    {
      // The array name starts with "Ion Mobility", but may carry additional
      // information such as the actual unit in which it was measured (seconds,
      // milliseconds, volt-second per square centimeter). We currently ignore
      // the unit but return the correct array.
      // For diaPASEF data converted with proteowizard ion mobility arrays are stored in "inverse reduced ion mobility"
      for (auto & bda : binaryDataArrayPtrs)
      {
        if (bda->description.find("Ion Mobility") == 0)
        {
          return bda;
        }
        else if (bda->description.find("mean inverse reduced ion mobility array") == 0)
        {
          return bda;
        }
      }
      return BinaryDataArrayPtr(); // return null
    }
```

Reply:
sorry, that is the correct one "mean inverse reduced ion mobility array"